pkg/block/azure/adapter.go

```diff
@@ -568,11 +568,11 @@ func (a *Adapter) CompleteMultiPartUpload(ctx context.Context, obj block.ObjectP
 
 func (a *Adapter) GetStorageNamespaceInfo() block.StorageNamespaceInfo {
 	info := block.DefaultStorageNamespaceInfo(block.BlockstoreTypeAzure)
-	info.ImportValidityRegex = `^https?://[a-z0-9_-]+\.(blob|adls)\.core\.windows\.net` // added adls for import hint validation in UI
+	info.ImportValidityRegex = `^https?://[a-z0-9_-]+\.blob\.core\.windows\.net`
```

Comment:
This breaks any existing import scripts.  Does this mean we need a major release 🤢 ?

Reply:
As stated in the PR description - this is a breaking change. But I believe we can avoid a major release since this affects specific users we can identify

Reply:
Added a WA for lakectl so it doesn't break. Effectively this is only going to impact WebUI users (I'm fine with that)